spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc

```diff
@@ -168,6 +168,24 @@ These properties can also be specified via `deployment` properties when deployin
 dataflow:>stream deploy foo --properties "app.*.logging.level.org.springframework.integration=DEBUG"
 ----
 
+==== Remote Debugging with Local Deployer
+
+When using Local Deployer you may want to benefit from the ability to debug your individual applications. This could be accomplished using remote debugging feature of the JVM.
```

Comment:
Same here, too. Language-wise, the SCDF users don't have to know the support in the deployer; the user-facing support is in SCDF (via deployer) anyway.